cpp/src/backends/CMakeLists.txt

```diff
@@ -27,28 +27,28 @@ set(TS_BACKENDS_TORCH_SCRIPTED_SOURCE_FILES "")
 list(APPEND TS_BACKENDS_TORCH_SCRIPTED_SOURCE_FILES ${TS_BACKENDS_TORCH_SCRIPTED_SRC_DIR}/torch_scripted_backend.cc)
 list(APPEND TS_BACKENDS_TORCH_SCRIPTED_SOURCE_FILES ${TS_BACKENDS_TORCH_SCRIPTED_SRC_DIR}/handler/base_handler.cc)
 add_library(ts_backends_torch_scripted SHARED ${TS_BACKENDS_TORCH_SCRIPTED_SOURCE_FILES})
-target_include_directories(ts_backends_torch_scripted PUBLIC 
+target_include_directories(ts_backends_torch_scripted PUBLIC
   ${TS_BACKENDS_TORCH_SCRIPTED_SRC_DIR} ${TS_BACKENDS_TORCH_SCRIPTED_SRC_DIR}/handler ${TORCH_INCLUDE_DIRS})
-target_link_libraries(ts_backends_torch_scripted PUBLIC ts_utils ts_backends_core ${TORCH_LIBRARIES}) 
+target_link_libraries(ts_backends_torch_scripted PUBLIC ts_utils ts_backends_core ${TORCH_LIBRARIES})
 install(TARGETS ts_backends_torch_scripted DESTINATION ${torchserve_cpp_SOURCE_DIR}/_build/libs)
 
 # build library ts_backend_torch_deploy
 #set(TS_BACKENDS_TORCH_DEPLOY_SOURCE_FILES "")
 #add_library(ts_backends_torch_deploy SHARED ${TS_BACKENDS_TORCH_DEPLOY_SOURCE_FILES})
 #target_include_directories(ts_backends_torch_deploy PUBLIC ${TS_BACKENDS_TORCH_DEPLOY_SRC_DIR})
```

Comment:
I have rebased my dev branch and removed all code changes to this file, but since I had touched this file in a previous commit, the [pre-commit hook](https://github.com/pytorch/serve/blob/be283977488ea186ac631634b2ed6a40f72a227d/.pre-commit-config.yaml#L17) to remove trailing whitespaces is still running on this file.